socioeconomic-differences/socioeconomic-differences.md

```diff
@@ -12,66 +12,88 @@ A person's socioeconomic status can inform their identity, as it affects the com
+       The following activity can help form base for the entire lesson.(10 min)
 
-### Common Mistakes / Misconceptions
+    3. Take the note of the points mentioned by the students and include the points in further lesson. (10 min)
+        Make sure to mention the following points:-
+          -Amidst their transformation journey(from blue-collar job to white-collar job)students may struggle with white-collar tacts such as:-
+	        -They may often feel out of place.
+            -They may try to outwork themselves in order to "fit" amongst their workspace.
+	        -Civil discussions about problems and solutions may feel like arguments.
+	        -It can be hard to accept compliments.
+	        -Their communities may scorn over their class straddling for it may be hard for them to accept the change too.
+	        -((Other problems can also be added as the discussion with students is continued.))
+	   
 
-List things that apprentices might not realize, might assume at first, or should avoid.
-
-- Example
-- Example
+    -The reasons why they may feel aloof or "out-of-place" are:-
+	   -Many such students support extended families as well.
+	   -They have learned to live with isolation.
+	   -They may not know the value of therapy.
+	   -Generally people often dress, eat, and drink differently i.e everybody has a different cultural and social background.
+     -They have scorned the rich and their lifestyle but now they are part of that atmosphere and it’s hard not to judge themselves or feel guilty.
+     -However,it is the instructor task to emphasize that it is okay to feel so and simultaneously provide valuable solutions to the students.
 
 
 ### Guided Practice
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
+    Do not spend a lot of time explaining each point, rather try to fit the points effectively within the given time frame by explaining each points briefly but thoroughly,however try your best to include the suggestions and remarks made by students during the course of discussion.
+    1. The participants may feel reluctant to initially share their experience.The instructor should try to encourage them kindly and carefully without asserting any obligation.
+    2. The instructor can recount the examples of local professionals in the area,who too had blue-collar lifestyle but successfully transitioned to white-collar job.Also do mention the struggles they faced(however do not repeat or elaborate problems mentioned above, rather keep it more personal and healthy discussion).
 
 
 ### Independent Practice
+    
+	1.The students can further read articles,books,listen to podcasts or watch videos to enhance their understanding of the subject.
+    
+### Challenge (2 hrs research after the class,preferably done in group)
```

Comment:
A 2 hour field trip is not going to happen, sorry.  Maybe as an ### Extensions section on line 94, you could suggest that if they have an informational interview with a Dev. Adv., they can ask these questions.

Reply:
In other words, please remove this independent practice section.